Makefile.third

```diff
@@ -6,7 +6,7 @@ fetchthirdparty:
 	git submodule init
 	git submodule sync
 	git submodule foreach --recursive git reset --hard
-	git submodule update
+	git submodule update --jobs 3 $(if $(CI),--depth 1,$(--recommend-shallow))
```

Comment:
Same remark as in https://github.com/koreader/koreader/pull/11677#discussion_r1564802473

It seems to me it's both more readable and friendlier to local git preferences to leave it empty, especially since it's the default.

```suggestion
	git submodule update --jobs 3 $(if $(CI),--depth 1,)
```